frontend/src/app/components/ExperimentPage.js

```diff
@@ -366,15 +372,27 @@ export default class ExperimentPage extends React.Component {
     });
   }
 
-  feedback() {
-    // Survey link is opened via href link in the template
+  feedback(evt) {
     sendToGA('event', {
       eventCategory: 'ExperimentDetailsPage Interactions',
       eventAction: 'button click',
-      eventLabel: 'Give Feedback'
+      eventLabel: 'Give Feedback',
+      outboundURL: evt.target.getAttribute('href')
+    });
+  }
+
+  showPreFeedbackDialog() {
+    this.setState({
+      showPreFeedbackDialog: true
     });
   }
 
+  handleFeedback(evt) {
+    evt.preventDefault();
+    const { pre_feedback_copy } = this.state.experiment;
+    (pre_feedback_copy === null || pre_feedback_copy === 'undefined') ? this.feedback(evt) : this.showPreFeedbackDialog();
```

Comment:
Should this be `typeof(pre_feedback_copy) === 'undefined'`? You could probably get away with just a `!pre_feedback_copy` unless we ever want the copy to literally be "0". Also should be a full if / else statement rather than a ternary condition, since we're not doing anything with the result value.